src/site/_transforms/responsive-images/index.js

```diff
@@ -0,0 +1,12 @@
+const cheerio = require("cheerio");
+const {determineImagePath} = require("./helpers");
+
+module.exports = (content, outputPath) => {
```

Comment:
So this won't run on partials. Would it be useful making this a shortcode instead, so we parse/update the content at the right stage of the pipeline? (i.e., at the same place we steal the partial right now)

I also don't think we should CDN images that are part of our site's UI.

Reply:
> So this won't run on partials. Would it be useful making this a shortcode instead, so we parse/update the content at the right stage of the pipeline? (i.e., at the same place we steal the partial right now)

I think you're referring to the service worker partials, and not nunjucks partials, correct?
I converted build-partial over to being a transform that will run after this transform.

> I also don't think we should CDN images that are part of our site's UI.
Why not?

Reply:
discussed over chat. Sam was concerned about CDN'ing these images as they're already cached by the SW and they're fundamentally different from content images. Sam was nice and let me keep it for now but we agreed if imgix goes down i will have to wear the shame cube.